thingif/src/main/java/com/kii/thingif/gateway/GatewayAddress.java

```diff
@@ -0,0 +1,73 @@
+package com.kii.thingif.gateway;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class GatewayAddress implements Parcelable {
```

Comment:
http://developer.android.com/intl/ja/reference/android/net/Uri.html

I think android's Uri class can be a candidate to a replacement of this class.